### DISCUSSION TITLE
Annual sponsorship tiers

### DISCUSSION THREAD

**Discussion by kepano** [community/community#11568] [Other Feature Feedback, Questions, & Ideas]
I would love to see GitHub Sponsors allow users to prepay for 1 year of sponsorship instead of monthly charges. Having a yearly membership would be tremendous.

**Comment by tohagan** [comment]
Rather painful at Tax time to have to convert 12 monthly receipts to local currency instead of a single annual receipt.

**Reply by MarkBennett** [reply]
Totally agree. It would be way easier to get my work to sponsor projects yearly, rather than by the month.

**Comment by TheMedo** [comment]
Same here, would love the option of annual payments mainly for tax and bookkeeping purposes.

**Comment by kdgyimah** [comment]
Agreed

How close are we on this?

**Comment by akirayamamoto** [comment]
I check all my debits on my statement every month, it would be easier if GH had the annual option.

**Comment by tohagan** [comment]
A single payment would likely lower transaction fees for GitHub and improve cashflow for everyone so surely a no-brainer Win-Win.

**Comment by tohagan** [comment]
For annual payments, I'd recommend sending a warning email 1 weeks prior so credit CC / billing details are up to date and the bill is not unexpected.

**Comment by Chizaruu** [comment]
Who said that there aren't annual payments already?
![image](https://user-images.githubusercontent.com/59434446/227808655-83822c8c-192b-4f7f-a748-745a707ca473.png)

Dw, I was also surprised when I saw $15 monthly turn into $180 yearly...

Sponsorships latch onto the account billing monthly/yearly method now. I wish I had known this before I sponsored. (Not saying the dev doesn't deserve it, but I'm budgeting ahh)

**Comment by fiver22** [comment]
Just tried to sponsor a project at the suggested level but there is only monthly available. Monthly debits are much harder for me to commit to than yearly -so my only option is a one time donation and a personal calendar reminder. It's not ideal.

**Comment by macolo** [comment]
Same here, annual would be preferred.

**Reply by macolo** [reply]
oh and PS - when a user has upgraded their account to annual payments, then shouldn't it say X USD per year instead of "X/12 USD per month" on a project's sponsoring page?

I have annual payments enabled in my personal github account but on the sponsoring page of, for example, https://github.com/sponsors/django-cms it still says "per month" which is not very accurate.

<img width="365" alt="image" src="https://github.com/community/community/assets/4056979/1c756690-1d84-4b23-8453-261a61fb17f8">

**Comment by AndrewTriesToCode** [comment]
Has GitHub responded at all to this request?

**Comment by gregsdennis** [comment]
Four years later.  Any updates?  I still don't see this as an option.

**Reply by gregsdennis** [reply]
Tack on another 18 months...
